docs/rules/configurable-rules.md

```diff
@@ -65,6 +65,8 @@ A minimum of one assertion property is required to be defined.
 ## Where object
 
 The `where` object is part of a `where` list which must be defined in order from the root node.
+A node cannot be used in the `where` list twice.
```

Comment:
Do you have a handy example to illustrate what you're trying to say here? I'm not sure I quite get it

Reply:
Yes, I do. 

A rule like the following won't work because of the duplicated `Schema` subject in the list:

```yaml
    subject: 
      type: Schema
    assertions: 
      required: 
        - minLength
        - maxLength
    where: 
      - subject: 
          type: Schema # <-- 
          property: type
        assertions:  
          const: string
      - subject: 
          type: Schema # <--
          property: enum
        assertions: 
          defined: false        

```

Instead, we have to group the assertions the other way:

```yaml
    subject: 
      type: Schema
    assertions: 
      requireAny: 
        - minLength
        - maxLength
        - enum
      mutuallyRequired: 
        - minLength
        - maxLength
    where: 
      - subject: 
          type: Schema # <-- used once in the list
          property: type
        assertions:  
          const: string
```

Reply:
I think I'm getting there. How about this:

```suggestion
Each node can only be used in one `where` objects for each assertion.
Rules that use multiple `where` objects must target each one on a different node.
```